monitors/filterWords.js

```diff
@@ -0,0 +1,42 @@
+exports.conf = {
+  enabled: true,
+  // Change to false if the filter should not ignore other bots
+  ignoreBots: true,
+  // Change to true if the filter should ignore selfBots
+  ignoreSelf: false,
+};
+
+exports.run = (client, msg) => {
+  // Please add the channel ID you want to recieve these reports in.
+  const modlog = "CHANNELIDHERE";
+  if (msg.deletable) {
+    const badWords = msg.guild.settings.swearWords.filter(word => msg.content.toLowerCase().includes(word));
+    if (badWords.length > 0) {
+      let badWordList = [];
+      for (let i = 0; i < badWords.length; i++) {
+        badWordList += badWords[i];
+      }
+      // DM the user regarding what words caused the message to be deleted.
+      msg.author.send(`I deleted the message below because you used a word that is not allowed. : **${badWordList.join(", ")}**`);
```

Comment:
Again, await. Also, you might want to send that in modlogs, not notify the user.

Reply:
This has to be to the user because it allows the user to know what message was deleted and the cause of the deletion. For example, I send a 1500 word announcement message but it has a bad word in there i didn't intend, it's good to receive a message saying hey this was deleted for this reason and here is the message so i can copy paste the deleted message and edit the necessary words and send it again